keps/sig-node/1948-add-dellocate-to-device-plugin-api/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Add Deallocate and PostStopContainer to device plugin API
+kep-number: 1948
+authors:
+  - "@mewais"
+owning-sig: sig-node
+participating-sigs: []
+status: implemented
```

Comment:
This should be provisional or implementable - implemented is for KEPs that have been approved and whose work is completed.